woocommerce/woocommerce
# Update Playwright to 1.32 and add command for UI mode

### All Submissions:






### Changes proposed in this Pull Request:

This PR updates Playwright to version 1.32 and adds a command to run Playwright e2e tests in UI mode -- `pnpm test:e2e-pw-ui`



### How to test the changes in this Pull Request:



1. Ensure that tests pass on CI
2. Pull this branch
3. Run `npx playwright install` to update Playwright
4. Run `pnpm env:start` (or `pnpm env:restart`) to start or reset the test environment.
5. Run `pnpm test:api-pw`.  Ensure that tests pass.
6. Run `pnpm env:restart` to reset the test environment.
7. Run `pnpm test:e2e-pw`. Ensure that tests pass.
8. Run `pnpm test:e2e-pw-ui`. Confirm that the Playwright  UI window is launched.




### Other information:




### FOR PR REVIEWER ONLY:
